README.md

```diff
@@ -10,46 +10,120 @@ It's important to note that the code in this repo may differ from the code writt
 ## Lab 00
 Running Fusion locally.
 
-1. Clone this repo
+1. Checkout your own branch
 ```
-git clone git@github.com:wapopartners/Fusion-Training-User-Stories.git
+git checkout USERNAME-fusion-training
 ```
 
-2. Change directory into Fusion-Training-User-Stories
+2. Install all packages/dependencies
 ```
-cd Fusion-Training-User-Stories
+npm install
 ```
 
-3. Checkout the `lab-00` branch
+3. To ensure you are using the latest fusion-cli, run:
 ```
-git checkout lab-00
+npm install @arc-fusion/cli@canary
 ```
 
-4. Checkout to your own branch from here
+4. Start fusion
 ```
-git checkout -b USERNAME-fusion-training
+npx fusion start
 ```
 
-5. Install all packages/dependencies
+5. Load the following link once fusion has finished loading (it might take a few minutes)
 ```
-npm install
+http://localhost/pagebuilder/pages
 ```
 
-6. To ensure you are using the latest fusion-cli, run:
+6. Note: you may see an alert warning of an error with content sources. Disregard this; we will address content sources later on.'
+
+7. Add the folders `mocks`, `components`, `content`, `environment`, and `properties`.
+
+8. In `mocks`, add the file `/user` with the following content:
+
 ```
-npm install @arc-fusion/cli@canary
+{
+   "displayName": "Local Developer",
+   "permissions": {
+     "PageBuilder": {
+       "global": [
+         "W",
+         "D",
+         "P",
+         "VRP",
+         "ERP",
+         "DT"
+       ]
+     }
+   }
+ } 
 ```
 
-7. Start fusion
+and the file `/siteservice/api/v3/website` with the following content:
+
 ```
-npx fusion start
+[
+  {
+    "_id": "demo",
+    "display_name": "Demo",
+    "is_default_website": true
+  }
+]
 ```
 
-8. Load the following link once fusion has finished loading (it might take a few minutes)
+9. In `components`, add the folders `output-types`, `layouts`, `chains`, and `features`
+
+10. In `content`, add the folders `schemas` and `sources`
+
+11. In `properties`, add the folder `sites`
+
+12. In the root director, add the file `.dockerignore` with the following content: 
```

Comment:
I believe this comes with npx fusion init or start

Reply:
I tried working through the steps and didn't see a .dockerignore

Reply:
I would leave this out for now. I do have it on all the repos I have compiled and never had to add it manually. Try running init then start and see if it creates it for you.